dbt/models/default/schema.yml

```diff
@@ -31,15 +31,15 @@ models:
             - pin
             - year
           config:
-            error_if: ">280662"
+            error_if: ">280679"
```

Comment:
As per usual these days, I also had to bump these thresholds to get tests to pass. Billy is still investigating but his hunch is that the assumptions behind these two tests are not actually correct.